release/cut-release-branch.sh

```
@@ -19,8 +19,6 @@ GIT_TOP_DIR=$(git rev-parse --show-toplevel)
 GIT_REMOTE=${GIT_REMOTE:-origin}
 GIT_BRANCH_TO_CUT_FROM=${GIT_BRANCH_TO_CUT_FROM:-viable/strict}
 
-# should output something like 1.11
-RELEASE_VERSION=${RELEASE_VERSION:-$(cut -d'.' -f1-2 "${GIT_TOP_DIR}/version.txt")}
 TEST_INFRA_BRANCH=${TEST_INFRA_BRANCH:-"release/${RELEASE_VERSION}"}
```

Here we should error out if RELEASE_VERSION is not set